site/_docs/history.md

```diff
@@ -29,9 +29,77 @@ Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
 {% comment %}
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.31.0">1.31.0</a> / TODO
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.32.0">1.32.0</a> / TODO
+{: #v1-32-0}
+
+#### Breaking Changes
+{: #breaking-1-32-0}
+
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using JDK/OpenJDK versions 8 to 18;
+Guava versions 19.0 to 31.1-jre;
+other software versions as specified in gradle.properties.
+
+#### New features
+{: #new-features-1-32-0}
+
+#### Bug-fixes, API changes and minor enhancements
+{: #fixes-1-32-0}
+
+#### Build and test suite
+{: #build-1-32-0}
+
+#### Dependency version upgrade
+{: #dependency-1-32-0}
+
+#### Web site and documentation
+{: #site-1-32-0}
+{% endcomment %}
+
```

Comment:
I don't thing we need to add entries for 1.32.0 at this point. It will be weird to have a big TODO element appearing first in the website. Please remove or comment out the changes around 1.32.0.